conf/broker.conf

```diff
@@ -1261,6 +1261,9 @@ gcsManagedLedgerOffloadReadBufferSizeInBytes=1048576
 # For more details, see the "Service Accounts" section of https://support.google.com/googleapi/answer/6158849
 gcsManagedLedgerOffloadServiceAccountKeyFile=
 
+# For Azure BlobStore ledger offload, Blob Container (Bucket) to place offloaded ledger into
```

Comment:
The configuration also works for aliyun
Would you please help add the configuration in `deployment/terraform-ansible/templates/broker.conf`? thanks.

Reply:
@hangc0276 and @MMirelli - Does this `managedLedgerOffloadBucket` configuration get used by more than Azure Blobstore? We should make sure the comment aligns with its usage. I'm not familiar enough with tiered storage configuration to know for sure.